content/docs/1.1.1/references/settings.md

```diff
@@ -328,25 +330,48 @@ This value can be lowered to avoid overprovisioning storage. See [Multiple Disks
 
 By disable replica rebuild, there won't be any new rebuild cross the whole cluster. The [Disks or Nodes Eviction Support](../../volumes-and-nodes/disks-or-nodes-eviction) and [Data Locality](../../high-availability/data-locality) feature won't work. But any restore features and currently rebuilding replica would still work as expected.
 
-#### Guaranteed Engine CPU
+#### Guaranteed Engine Manager CPU
+
+> Default: `12`
 
-> Default: `0.25`
+This integer value indicates how many percentage of the total allocatable CPU on each node will be reserved for each engine manager Pod. For example, 10 means 10% of the total CPU on a node will be allocated to each engine manager pod on this node. This will help maintain engine stability during high node workload.
+
+In order to prevent unexpected volume engine crash as well as guarantee a relative acceptable IO performance, you can use the following formula to calculate a value for this setting:
```

Comment:
"In order to prevent an unexpected volume engine crash as well as guarantee a relatively acceptable I/O performance..."